Development/v4.x/configuration.md

```diff
@@ -7,7 +7,10 @@
 
 ## SciCat Frontend
 
-This is the configuration file for the frontend client. The configuration file allows the systems administrator to configure every aspect of the client including switching on/off almost all non essential features. The configuration file can either be served from the backend, via the `/client/config.json` endpoint or mounted into `/usr/share/nginx/html/assets/config.json`.
+This is the configuration file for the frontend client. The configuration file allows the systems administrator to configure every aspect of the client including switching on/off almost all non essential features. 
+
+The configuration file can be served from the backend, via the `admin/client/config.json` endpoint. The frontend would expect the backend to be served on port 3000 in a routing configuration for this  to work.  The configuration file can also be mounted to the volume `/home/node/app/dist/config/frontend.config.json` for it to be served through this endpoint. 
```

Comment:
I'm not sure at this point that I can recommend serviing if from the frontend, based on previous discussions on [this issue](https://github.com/SciCatProject/frontend/issues/1186)

Reply:
This is an interesting point, at the moment I just want some documentation on how one can set up v4 with the frontend in its current state. What would we have to change in SciCat in order to server the config in a more intuitive way? Could this be another PR into the documentation when changes to how the frontend config is served have been made?